inference-engine/samples/hello_classification/main.cpp

```diff
@@ -130,6 +130,7 @@ int main(int argc, char *argv[]) {
 
         // --------------------------- 8. Process output ------------------------------------------------------
         Blob::Ptr output = infer_request.GetBlob(output_name);
+        if (output->size() != 1) throw std::logic_error("Sample supports topologies with 1 output only");
```

Comment:
I suppose `output->size()` defines a number of classes, is not it?

Reply:
Yes I agree that check is incorrect.

If you want to check that topology has only one output you need to check the number of outputs. At the current moment you check the size of output blob.

Reply:
Yes, sorry, this check is incorrect. I updated check. @ilya-lavrenov , @ilyachur  can you look at new version?